react/features/token-data/actions.js

```diff
@@ -29,8 +29,14 @@ export function parseTokenData() {
         // the iss field is not anonymous. If you want to pass data with
         // JWT token but you don't want to pass the JWT token for
         // verification the iss field should be set to "anonymous".
-        if (!config.token && payload && payload.iss !== 'anonymous') {
-            config.token = token;
+        if (payload && payload.iss !== 'anonymous') {
+            parsedToken.token = token;
```

Comment:
I don't get it. "token" is the "jwt" parameter which we are passing to TokenData. This in turn will get stored in a "jwt" instance variable. So parsedToken.token and parsedToken. jwt will be the same, sometimes.